First_section.html

```diff
@@ -187,21 +196,160 @@ <h1 class="project_name">Multi-Post Stories</h1>
+                <li class="ht3"><a class="css" href="#">css</a></li>
+                <li class="ht4"><a class="javascript" href="#">javascript</a></li>
+             </ul>
+            <div class="link_to_project">
+              <a class="check_Project" href="#">See Project</a>
+            </div>
+          </div>
+        </div>
+
+        <div class="work works2">
+        <img class="firstimage" src="Snapshoot Portfolio (6).png" alt="snashoot" />
+          <div class="Project1">
+            <h2 class="project_name">Facebook 360</h2>
+            <ul class="client">
+              <li id="h3">FACEBOOK</li>
+              <li class="Counter">
+                <img src="Counter.png" alt="counter" />
+              </li>
+
+              <li class="langue">Full Stack Dev</li>
+              <li class="Counter">
+                <img src="Counter.png" alt="counter" />
+              </li>
+
+              <li class="year">2015</li>
+            </ul>
+            <p class="project_description">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+            <ul class="code_language">
+              <li class="ht1"><a class="html" href="#">html</a></li>
+              <li class="ht2"><a class="ruby" href="#">Ruby on rails</a></li>
+              <li class="ht3"><a class="css" href="#">css</a></li>
+              <li class="ht4"><a class="javascript" href="#">javascript</a></li>
+            </ul>
+            <div class="link_to_project">
+              <a class="check_Project" href="#">See Project</a>
+            </div>
+          </div>
+        </div>
+
+        <div class="work works3">
+        <img class="secondimage" src="Snapshoot Portfolio (7).png" alt="snashoot" />
+          <div class="Project2">
+            <h1 class="project_name">Uber Navigation</h1>
+            <ul class="client">
+                <li id="h3">Uber</li>
+              <li class="Counter">
+                <img src="Counter.png" alt="counter" />
+              </li>
+
+              <li class="langue">Lead Developer</li>
+              <li class="Counter">
+                <img src="Counter.png" alt="counter" />
+              </li>
+
+              <li class="year">2018</li>
+            </ul>
+            <p class="project_description">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+            <ul class="code_language">
+              <li class="ht1"><a class="html" href="#">html</a></li>
+              <li class="ht2"><a class="ruby" href="#">Ruby on rails</a></li>
+              <li class="ht3"><a class="css" href="#">css</a></li>
+              <li class="ht4"><a class="javascript" href="#">javascript</a></li>
+            </ul>
+            <div class="link_to_project">
+              <a class="check_Project" href="#">See Project</a>
```

Comment:
- [ ] Kindly add button interaction styles as shown in Figma to all the buttons, some buttons don't have the styling and others have the wrong one (the form submission button for example). Kindly fix the issue 💯 

## Your design:
![image](https://user-images.githubusercontent.com/50721479/207922204-3cb62413-ff2e-45ab-b702-c5a81c57894b.png)

## Figma design:
![image](https://user-images.githubusercontent.com/50721479/207922155-6d032ea3-3bc4-4374-8e92-7d46e3e2b3f4.png)

Reply:
I applied the hover and chose prefered background color. I appreciate it.